src/components/Form/Input/index.tsx

```diff
@@ -1,29 +1,35 @@
-import React from 'react'
-
+import React from 'react';
 
 interface InputProps {
-    value: string
-    setValue: React.Dispatch<React.SetStateAction<string>>
-    className?: string
-    label?: {
-        text: string
-        id: string
-    }
-    placeholder?: string
+	value: string;
+	setValue: React.Dispatch<React.SetStateAction<string>>;
+	className?: string;
+	label?: {
+		text: string;
+		id: string;
+	};
+	placeholder?: string;
```

Comment:
```suggestion
	placeHolder?: string;
```

Po, melhor com letra maiuscula pra seguir o template do restante das variaveis.